src/PostgREST/Query/SqlFragment.hs

```diff
@@ -438,6 +441,12 @@ responseHeadersF = currentSettingF "response.headers"
 responseStatusF :: SQL.Snippet
 responseStatusF = currentSettingF "response.status"
 
+pgrstInsertedF :: SQL.Snippet
+pgrstInsertedF = "count(nullif(current_setting('pgrst.mutation'), 'inserted'))"
```

Comment:
Hmm... I think this `current_setting('pgrst.mutation')` only returns the last value that was set here? Or does it compare with every insert/update?

Also the `nullif(..., 'inserted')` means that it will turn all 'inserted' into `null` and won't count it, so it should be in `pgrstUpdatedF` instead (and vice versa).

Anyways, this would change with the suggestion I gave in the other comment.